.stutter/java8.lock

```diff
@@ -1,5 +1,8 @@
 # DO NOT MODIFY: Generated by Stutter plugin.
+4.9
+4.10.3
 5.1.1
 5.6.4
 6.0.1
 6.5
+6.6-milestone-1
```

Comment:
I manually removed this one last time I updated the files. Given it is not a final release and each Gradle version substantially increases our build time, I think we could leave it out for now